tests/test_matching_classes.py

```diff
@@ -0,0 +1,216 @@
+import pytest
+import rdflib
+from rdflib.namespace import OWL, RDF, RDFS
+from rdflib import Graph, URIRef
+import json
+import brickschema
+import sys
+
+sys.path.append("..")
+
+from bricksrc.version import BRICK_VERSION  # noqa: E402
+from bricksrc.namespaces import BRICK  # noqa: E402
+
+
+def getDict(g, q):
+    dict = {}
```

Comment:
We should avoid using builtin keywords as variable names; change this to `d` or something that's not a keyword

Reply:
Agreed. Changed accordingly.